client/state/purchases/test/reducer.js

```diff
@@ -33,27 +33,29 @@ describe( 'reducer', () => {
 	} );
 
 	test( 'should return an object with the list of purchases when fetching completed', () => {
+		const siteId2 = '2828828282';
 		let state = reducer( undefined, {
 			type: PURCHASES_USER_FETCH_COMPLETED,
 			purchases: [
-				{ ID: '1', blog_id: siteId, user_id: userId },
+				{ ID: '1', blog_id: siteId2, user_id: userId },
```

Comment:
If we don't change the site for id 1, then it will be removed by the site fetch which does not return id 1.